src/alpha.c

```diff
@@ -198,7 +194,7 @@ avifResult avifRGBImagePremultiplyAlpha(avifRGBImage * rgb)
                 }
                 row += rgb->rowBytes;
             }
-        } else {
+        } else if (rgb->format == AVIF_RGB_FORMAT_ARGB || rgb->format == AVIF_RGB_FORMAT_ABGR) {
```

Comment:
I listed the supported RGB formats explicitly and avoided using `else` for "the rest of the formats with alpha". This makes the code more robust if we add a new RGB format with alpha in the future.